docker/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
this file needs to have `0755` permissions rather than `0644`

Reply:
Removed this file based on offline discussion